src/plugins/intel_gpu/src/graph/include/reshape_inst.h

```diff
@@ -42,8 +43,12 @@ struct typed_program_node<reshape> : public typed_program_node_base<reshape> {
         if (!input().is_type<crop>())
             return false;
 
-        // TODO: If user is RoPE and dynamic padding exists, ouput padding propagation is not supported in the base mode
-        if (get_users().size() == 1 && get_users().front()->is_type<rope>())
+        // oneDNN supports padded input of outer axis only for buffer fusing on static shape
+        if (!has_outer_padding_offset() && get_users().front()->get_preferred_impl_type() == impl_types::onednn)
+            return false;
+
+        // TODO: If user is RoPE or MVN and dynamic padding exists, ouput padding propagation is not supported in the base mode
+        if (get_users().size() == 1 && (get_users().front()->is_type<rope>() || get_users().front()->is_type<mvn>()))
```

Comment:
Is this because rope and mvn does not support dynamic padding?

Reply:
Yes. This is one of the causes of the issue.
Dynamic padding is not supported for data indexing for MVN's fused ops.

Reply:
After offline discussion: TODO get a more general solution (e.g., node not supporting padding..)